src/Stars.cpp

```diff
@@ -544,16 +538,15 @@ void Stars::init(const std::string& sStarTextureFile, const std::string& sCacheF
         if (mCatalogs.find(CatalogType::eTycho) == mCatalogs.end()) {
           readStarsFromCatalog(it->first, it->second);
         } else {
-          std::cout << "[Stars] Failed to load Tycho2 catalog: "
-                    << "Tycho already loaded!" << std::endl;
+          spdlog::error("Failed to load Tycho2 catalog: Tycho already loaded!");
```

Comment:
Shouldn't that be a warning? This looks to be recovarable.
```suggestion
          spdlog::warn("Failed to load Tycho2 catalog: Tycho already loaded!");
```